include/plugin/settings.h

```diff
@@ -26,6 +26,7 @@ struct Settings
     std::vector<double> max_flow_acceleration;
     std::vector<double> layer_0_max_flow_acceleration;
     std::vector<double> gradual_flow_discretisation_step_size;
+    double reset_flow_duration;
```

Comment:
Initialize the value, with the vector that is not needed but with a single value it is good practice
```suggestion
    double reset_flow_duration { 0.0 };
```